irohad/ametsuchi/impl/storage_impl.cpp

```diff
@@ -272,20 +254,58 @@ namespace iroha {
       return wsv_command.insertPeer(peer);
     }
 
+    expected::Result<std::unique_ptr<MutableStorage>, std::string>
+    StorageImpl::createMutableStorageWithoutBlockStorage() {
+      return createMutableStorage([] {
+        struct BlockStorageStub : public BlockStorage {
+          bool insert(std::shared_ptr<const shared_model::interface::Block>
+                          block) override {
+            return true;
```

Comment:
This `true` could be misleading. I suggest adding an explanation about the inconsistency of implementation: insert returns true, but fetch returns none.